### PR TITLE
debt: show dialog for error message in settings screen of demo app

### PR DESCRIPTION
**Pull request by khairul-alam-licon**
# Description
As a demo app user, I want to take screenshot easily when there is any error shown in settings screen specially while there is `MiniAppSdkException`. Previously, there was a `Toast` which disappears after a moment, so hard to take screenshot. I may also want to copy those error messages to share. 

# Screenshots 
<table><tr><td>
    
</td>
<td>
</td>
<td>
     
</td>
</tr>
</table>

# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

**Review by corycaywood — APPROVED** [review]
LGTM 👍

File: testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/userdata/ProfileSettingsActivity.kt [hunk 1]
```diff
@@ -31,8 +29,6 @@ class ProfileSettingsActivity : BaseActivity() {
     private lateinit var profileUrl: String
     private lateinit var profileUrlBase64: String
     private lateinit var binding: ProfileSettingsActivityBinding
-    private val coroutines = MiniAppCoroutines()
```
Comment (khairul-alam-licon) [inline testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/userdata/ProfileSettingsActivity.kt:34]: Sorry, I just realized that we don't actually need it as `BaseActivity` already has a similar support. So, I cleaned up a bit. I have tested it, it's working fine.

File: testapp/src/main/java/com/rakuten/tech/mobile/testapp/helper/AppHelper.kt [hunk 2]
```diff
@@ -24,3 +32,33 @@ fun parseStringToDate(format: String, str: String): Date {
     }
     return Date()
 }
+
+fun showNormalDialog(activity: Activity, content: String) {
```
Comment (minh-rakuten) [inline testapp/src/main/java/com/rakuten/tech/mobile/testapp/helper/AppHelper.kt:36]: `showDialog` is normal enough to me 😆

**Review by minh-rakuten — APPROVED** [review]
👍

**Pull request merged**